src/lib/components/card/Card.component.js

```diff
@@ -0,0 +1,134 @@
+import type { Node, HTMLProps } from 'react';
+import React from 'react';
+import styled from 'styled-components';
+import { getTheme, hex2RGB } from '../../utils';
+
+const CardHeader = styled.div`
+  padding: 0.7rem;
+  border-radius: 3px;
+  font-weight: bold;
+`;
+
+const CardBody = styled.div`
+  color: ${(props) => getTheme(props).textPrimary};
+  height: 100%;
+`;
+
+const CardContainer = styled.div`
+  border-radius: 3px;
+  display: flex;
+  flex-flow: column;
+  width: ${(props) => props.width};
+  height: ${(props) => props.height};
+  background: ${(props) =>
+    getTheme(props)[props.bodyBackgroundColor || 'backgroundLevel3']};
+
+  .${CardBody.styledComponentId} {
+    height: 100%;
+    position: relative;
+    opacity: ${(props) => (props.disabled ? '0.2' : '1')};
+  }
+
+  .${CardHeader.styledComponentId} {
+    border-radius: 2px;
+    color: ${(props) => getTheme(props).textPrimary};
+    ${(props) => {
+      const hexColor = hex2RGB(
+        getTheme(props)[
+          props.status || props.headerBackgroundColor || 'backgroundLevel4'
+        ],
+      ).join(',');
+      let opacity = 1;
+      if (props.status && !props.disabled) opacity = 0.4;
+
+      return `background: rgba(${hexColor}, ${opacity});`;
+    }}
+    ${(props) => props.disabled && 'opacity: 0.3;'}
+  }
+
+  ${(props) =>
+    props.onClick && !props.disabled
+      ? `
+      cursor: pointer;
+  
+      &:hover {
+        box-shadow: 0 0 0 2px ${getTheme(props).highlight};
+      }
+  
+      &:focus {
+        outline: 0;
+        box-shadow: 0 0 0 1px ${getTheme(props)[props.status || 'highlight']};
+      }
```

Comment:
@Cuervino what do you think of this `focus` state for the cards ?

Reply:
IMO, the focus state like that is too close to the hover state.
There should be a "additive" logic.
I think that an outline 2 px out of the card with the color buttonSecondary should be good.
<img width="166" alt="navigation-status=Focus, Health=healthy" src="https://user-images.githubusercontent.com/28654802/118679684-5de81880-b7fe-11eb-8556-be70294198c9.png">